src/extensions/hooks/adopted-probe/src/validate-fields.ts

```diff
@@ -47,7 +47,11 @@ export const validateTags = async (fields: Fields, keys: string[], accountabilit
 	const tagsSchema = Joi.array().items(Joi.object({
 		value: Joi.string().trim().pattern(/^[a-zA-Z0-9-]+$/).max(32).required(),
 		prefix: Joi.string().valid(...validPrefixes).required(),
-	})).max(5);
+	}));
+
+	if (fields.tags.length > 5) {
+		throw payloadError('A maximum of 5 tags is allowed.');
+	}
```

Comment:
I see you picked the current max used value, but I'd definitely like to set this higher if possible (8 or even 16) to make it very unlikely users even hit it. The index field is currently set to 4090 size, with 100 characters at most per tag, it seems unlikely we'd exceed that? And since it's varchar, it could be declared to be even larger if needed.

Reply:
Ok, I misinterpreted this at first, now I see the limitation has always been there, and we just had wrong seed data. Let's keep it as it is, then.

Reply:
github username limit is 39, and tag value limit is 32, plus a few more for optional `format` and JSON characters. 16 tags will fit easily.